Tests/Realm.Tests/Sync/SessionTests.cs

```diff
@@ -828,25 +837,18 @@ public void Session_OnSessionError()
                             }
 
                             completionTcs.TrySetResult();
+                            lastReportedProgress = p.ProgressEstimate;
```

Comment:
Why is this happening after we complete the tcs?

Reply:
It shouldn't, it should happen all the time, I'll fix it 🤦